content/en/security/infrastructure_vulnerability/setup.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Setting up Infrastructure Vulnerability Management 
+kind: documentation
+is_beta: true
+further_reading:
+- link: "/security/infrastructure_vulnerability/"
+  tag: "Documentation"
+  text: "Infrastructure Vulnerability Management"
+---
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">
+Infrastructure Vulnerability Management is not available on the US1-FED Datadog site.
+</div>
+{{< /site-region >}}
+
+## Pre-requisites
+
+- [Cloud Security Management][1] (using [Cloud Workload Security][2] or [Cloud Security Posture Management][3]).
+- Datadog Agent 7.44.0 or higher.
+- [containerd][4] 1.5.6 or higher.
+- [Helm Chart][5] 3.22.0 or higher.
+- 5GB or more of free disk space on the Datadog Agent host. 
+
+**Note**: In order for the Datadog Agent to scan for vulnerabilities, the free disk space must be equivalent to the size of your largest container image. If you have container images larger than 5GB, you would need to have additional equivalent free disk space on the host. 
+
+## Setup
+
+Add the following options to your `values.yaml` configuration file:
+
+**Note**: This enables the Container Image Metadata collection (allowing you to explore container images) and SBOM collection in your Datadog Agent, allowing you to scan libraries present in container images and detect vulnerabilities.
```

Comment:
**Note**: This enables the Container Image Metadata collection (allowing you to explore container images) and SBOM collection in your Datadog Agent (allowing you to scan libraries present in container images and detect vulnerabilities).